tf2/src/buffer_core.cpp

```diff
@@ -1666,31 +1672,34 @@ void BufferCore::_chainAsVector(const std::string & target_frame, TimePoint targ
         assert(0);
       }
     }
-    int m = (int)target_frame_chain.size()-1;
-    int n = (int)source_frame_chain.size()-1;
-    for (; m >= 0 && n >= 0; --m, --n)
```

Comment:
Same here. I'd rather be cautious and not change the logic in this PR.

Reply:
See https://github.com/ros2/geometry2/pull/270#discussion_r434196136.